schemas/context/profile-privacy.example.1.json

```diff
@@ -2,11 +2,74 @@
 	"xdm:optOutConsentLevel": {
 		"xdm:privacyOptOuts": [
 			{
-			"xdm:optOutType": "general_opt_out",
-			"xdm:optOutValue": "out",
-			"xdm:timestamp": "2019-01-01T15:52:25+00:00"
+				"xdm:optOutType": "general_opt_out",
+				"xdm:optOutValue": "in",
+				"xdm:timestamp": "2019-01-01T15:52:25+00:00",
+				"xdm:optOutBasisOfProcessing": "legitimate_interest"
+			},
+			{
+				"xdm:optOutType": "device_linking",
+				"xdm:optOutValue": "not_provided",
+				"xdm:optOutBasisOfProcessing": "vital_interest"
+			},
+			{
+				"xdm:optOutType": "anonymous_analysis",
+				"xdm:optOutValue": "out"
 			}
-		]
+		],
+		"xdm:personalizationPreferences": {
+			"xdm:defaultPersonalizationPreference": {
+				"xdm:defaultPersonalizationChoice": "unknown",
+				"xdm:defaultPersonalizationTimestamp": "2019-01-01T15:52:25+00:00",
+				"xdm:defaultPersonalizationBasisOfProcessing": "consent"
+			},
+			"xdm:personalizationOptions": [
+				{
+					"xdm:personalizationType": "email",
+					"xdm:personalizationChoice": "in"
+				},
+				{
+					"xdm:personalizationType": "push_notifications",
+					"xdm:personalizationChoice": "out",
+					"xdm:personalizationBasisOfProcessing": "legitimate_interest"
+				}
+			]
+		},
+		"xdm:marketingPreferences": {
+			"xdm:defaultMarketingPreference": {
+				"xdm:defaultMarketingChoice": "unknown"
+			},
+			"xdm:marketingOptions": [
+				{
+					"xdm:marketingType": "email",
+					"xdm:marketingChoice": "in",
+                    "xdm:marketingSubscriptions": {
+                       "weekly_mailer": {
```

Comment:
this isn't a valid XDM key name.  It needs to be namespaced and camelCased.

Reply:
This "xdm:subscriptions" is defined as a "Map" datatype, so this is example is showing a potential key in that map. In XDM (and they way AEP uses Maps) the key is simply a "string". The "value" portion of that Map field has a schema in which case those fields are namespaced and defined as needed.